src/Npgsql/NpgsqlSlimDataSourceBuilder.cs

```diff
@@ -355,7 +355,16 @@ public NpgsqlSlimDataSourceBuilder EnableRecords()
     /// </summary>
     public NpgsqlSlimDataSourceBuilder EnableFullTextSearch()
     {
-        // AddTypeInfoResolver(new FullTextSearchTypeInfoResolver());
+        AddTypeInfoResolver(new FullTextSearchTypeInfoResolver());
+        return this;
+    }
+
+    /// <summary>
+    /// Sets up mappings for extra conversions from PostgreSQL to .NET types.
```

Comment:
Nice and vague 😅 maybe mention numeric conversions?

Reply:
We could slip in other conversions :) I don't really care, feel free to edit if you want.